### DISCUSSION TITLE
import question

### DISCUSSION THREAD

**Discussion by qziov** [pyscript/pyscript#1918] [Q&A]
when I import pandas ，it callback this
![Snipaste_2024-01-02_17-14-32](https://github.com/pyscript/pyscript/assets/103866547/d2b81789-b325-4e2a-b18c-f45a9d02da04)

this my code
![Snipaste_2024-01-02_17-15-40](https://github.com/pyscript/pyscript/assets/103866547/7ff49999-f8ed-40e5-9d1b-de8732f95368)
![Snipaste_2024-01-02_17-15-10](https://github.com/pyscript/pyscript/assets/103866547/b6f3aacd-1ea9-4db2-9a9e-ed459ec607a1)

![Snipaste_2024-01-02_17-15-20](https://github.com/pyscript/pyscript/assets/103866547/65bb6870-5876-48f2-9770-4560968a0acd)

**Comment by WebReflection** [comment]
I think you are not using the latest PyScript release ... please update your HTML to use latest version and use a module for the core functionality, as explained in the documentation: https://pyscript.github.io/docs/2023.12.1/beginning-pyscript/

**Reply by qziov** [reply]
when I use `<script type="module" src="https://pyscript.net/releases/2023.12.1/core.js"></script>`
It traceback another Error
ValueError: Can't find a pure Python 3 wheel for: 'pandas>=2.0.0'
![Snipaste_2024-01-03_10-35-06](https://github.com/pyscript/pyscript/assets/103866547/fbec7e27-52b2-42c5-8f53-981c4eafc8a9)

**Reply by qziov** [reply]
I maybe known what happen. 
Maybe is flopy，it can not use pandas>=2.0.0
